composer.json

```diff
@@ -13,7 +13,9 @@
   "require": {
     "php": "^8.3",
     "laravel/framework": ">=11.34",
-    "lcobucci/jwt": "^5.3"
+    "lcobucci/jwt": "^5.3",
+    "winter/laravel-config-writer": "^1.2",
+    "ronasit/laravel-helpers": "^3.5"
```

Comment:
if the helpers package required only for the test - please move it to the `require-dev` block